scripts/tasks/src/lint-imports.ts

```diff
@@ -200,17 +201,15 @@ function lintImports(
     }
   }
 
-  /**
-   * @param importMatch - Result of running `importStatementRegex` against a single import
-   * (`[1]` will be the import path)
-   */
-  function _evaluateImport(
-    filePath: string,
-    importMatch: RegExpMatchArray,
-    importErrors: ImportErrors,
-    isExample?: boolean,
-  ) {
-    const importPath = importMatch[2];
+  // eslint-disable-next-line @typescript-eslint/no-shadow
+  function _evaluateImport(options: {
```

Comment:
refactored to options pattern - having more arity >2 of an function API is a bad practice/code smell (makes the function definition unreadable - easy to pass wrong arg on wrong place)